READMEs/c2d-flow.md

```diff
@@ -280,8 +302,15 @@ print(f"Started compute job with id: {job_id}")
 In the same Python console, you can check the job status as many times as needed:
 
 ```python
-ocean.compute.status(DATA_did, job_id, bob_wallet)
-
+# Wait until job is done
```

Comment:
I don't want to add this here, this is a README and people will run it from the console. Usually they will just try the status a few times until they get the correct status, not programatically _wait_ for it. @trentmc what do you think?

Reply:
Summary: I recommend that we keep the "wait loop".

Details...

I used to also have the opinion to "not programatically wait for it" // have a wait loop. However, we saw in another README elsewhere during 2021 with "wait for it" that people would _not_ wait for it (because they were copy-pasting), so they wouldn't get their results back. Then they'd go to discord or otherwise and say it's broken. This happened at least 5 times in a couple months. So we fixed it by making it where it would wait, mostly by putting blocking into ocean.py. Then the problem went away.

Another reason to keep a "wait loop" is that people _will_ be copying & pasting these snippets into their own code, where they do need it to wait. Use case: most production uses of C2D. So, having code ready as a good default will reduce friction for users.